client/src/components/Ticketing/ticketingmanager/Season/SeasonInstancesPage.tsx

```diff
@@ -0,0 +1,134 @@
+import React, {useEffect, useState} from 'react';
+import {useNavigate} from 'react-router-dom';
+import {titleCase} from '../../../../utils/arrays';
+import {useFetchToken} from '../showings/ShowingUpdated/ShowingUtils';
+import {getSeasonImage, formatSeasonDate} from './seasonUtils';
+
+export interface Seasons {
+  seasonid: number;
+  name: string;
+  startdate: number;
+  enddate: number;
+  imageurl?: string;
+}
+
+/**
+ * Display all seasons page
+ *
+ * @module
+ * @returns Season Instances Page
+ */
+const SeasonInstancesPage = () => {
+  const navigate = useNavigate();
+  const [seasons, setAllSeasons] = useState<Seasons[]>([]);
+  const {token} = useFetchToken();
+
+  const getAllSeasons = async () => {
+    try {
+      const getAllSeasonsRes = await fetch(
+        process.env.REACT_APP_API_2_URL + '/season',
+        {
+          credentials: 'include',
+          method: 'GET',
+          headers: {
+            'Content-Type': 'application/json',
+            'Authorization': `Bearer ${token}`,
+          },
+        },
+      );
+
+      if (!getAllSeasonsRes.ok) {
+        throw new Error('Failed to fetch all seasons');
+      }
+
+      const seasonsData = await getAllSeasonsRes.json();
+      setAllSeasons(seasonsData);
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  useEffect(() => {
+    getAllSeasons();
+  }, [token]);
+
+  return (
+    <div className='w-full h-screen overflow-x-hidden absolute'>
+      <div
+        className='md:ml-[18rem] md:mt-40 sm:mt-[11rem]
+       sm:ml-[5rem] sm:mr-[5rem] sm:mb-[11rem] h-full'
+      >
+        <section className={'flex flex-row justify-between w-full mb-14'}>
+          <h1
+            className='col-span-2 min-[678px]:col-span-1 font-bold text-5xl bg-clip-text
+           text-transparent bg-gradient-to-r from-sky-500
+            to-indigo-500'
+          >
+            Select Season
+          </h1>
+          <button
+            onClick={() => navigate(``)} // TODO, create seasons page
+            className={
+              'bg-green-500 hover:bg-green-700 h-fit disabled:bg-gray-500 text-white p-2 font-bold rounded-xl flex flex-row'
+            }
+          >
+            <svg
+              xmlns='http://www.w3.org/2000/svg'
+              fill='none'
+              viewBox='0 0 24 24'
+              strokeWidth={3}
+              stroke='currentColor'
+              className='w-[2rem] h-[2rem] pr-1'
+            >
+              <path
+                strokeLinecap='round'
+                strokeLinejoin='round'
+                d='M12 6v12m6-6H6'
+              />
+            </svg>
+            <span className={'my-auto text-lg'}>Create Season</span>
+          </button>
+        </section>
+        <ul className='md:grid md:grid-cols-2 md:gap-8 sm:grid sm:grid-cols-1 sm:gap-4 mt-9'>
```

Comment:
You can remove any instance of the `sm:` breakpoint from your code since `sm` is the smallest breakpoint and we don't typically need the reference anything smaller than that size

Reply:
Got it, thanks!